src/dawn/SIR/ASTExpr.h

```diff
@@ -402,6 +402,8 @@ class VarAccessExpr : public Expr {
   const std::string& getName() const { return name_; }
   std::string& getName() { return name_; }
 
+  void setName(std::string name) { name_ = name; }
```

Comment:
if we have a setter, we should avoid the non-const getter

Reply:
good point. Done